src/pages/AddLiquidity/index.tsx

```diff
@@ -405,9 +407,11 @@ export default function AddLiquidity({
                           width={approvalB !== ApprovalState.APPROVED ? '48%' : '100%'}
```

Comment:
this button should probably be pulled out into a component

Reply:
I haven't read this code carefully. Is there anything I need to help optimize?